docs/schema.md

```diff
@@ -1055,6 +1055,51 @@ potential encoding of a vulnerability entry.
 }
 ```
 
+## Debian vulnerability
+
+Debian security advisory (DSA) does not use this format currently, but using the tool located in `/tools/debian/`, 
```

Comment:
nit: instead of specifying `/tools/debian`, link to a permalink (i.e. with commit hash) for the directory.